ogr/abstract.py

```diff
@@ -504,12 +503,6 @@ class MergeCommitStatus(Enum):
 
 
 class PullRequest(OgrAbstractClass):
-    @deprecate_and_set_removal(
```

Comment:
Oh, I know… The PR constructor should be same as the ones for other classes, so:
```py
def __init__(self, raw_pr: Any, project: "GitProject") -> None:
    ...
```

And the current way should be **only** in the read-only pull request class. We didn't switch, because
1. it wasn't backward compatible
2. we didn't have it fixed even in our own codebase…

I would suggest you to merge the PR you have open in service, cause this change can probably break some tests again, so you have as few errors as possible. And then also the properties need to be updated :/

---

Context: https://github.com/packit/ogr/pull/287/files#diff-6a8c9031cacfc4f2666634c0abfa043e64080b713a5957e8ba5b36e06395d674L274

Reply:
Yes, I've already merged it, thanks for the info. So should we try to do the backward incompatible change now, since we are nearing 1.0.0 where it should be fine to make the change? Are there going to be some releases between now and 1.0.0? If so, then I'd suggest postponing merging this PR and making the changes just into 1.0.0, not 0.X.

Reply:
doesn't really matter, they should've been removed already :)